cluster/gce/gci/configure-helper.sh

```diff
@@ -2155,6 +2155,12 @@ EOF
     update-event-exporter ${event_exporter_yaml}
     fluentd_gcp_version="${FLUENTD_GCP_VERSION:-0.2-1.5.28-1}"
     sed -i -e "s@{{ fluentd_gcp_version }}@${fluentd_gcp_version}@g" "${fluentd_gcp_yaml}"
+    if [[ "${LOGGING_STACKDRIVER_RESOURCES:-}" == "new" ]]; then
```

Comment:
I think we need to add this to PROVIDER_VARS so that it is set in the cluster.

Reply:
I don't see `HEAPSTER_USE_OLD_STACKDRIVER_RESOURCES` or `HEAPSTER_USE_NEW_STACKDRIVER_RESOURCES` in the `PROVIDER_VARS`, are you sure we need it, if we do, are we missing it for these heapster variables?